ui/js/filters.js

```diff
@@ -50,11 +50,6 @@ treeherder.filter('getRevisionUrl', function () {
     };
 });
 
-//http://stackoverflow.com/questions/16630471/how-can-i-invoke-encodeuricomponent-from-angularjs-template
-treeherder.filter('encodeURIComponent', function () {
```

Comment:
The test for this can be removed now:
https://github.com/mozilla/treeherder/blob/3be9568cbab021d4b934bb31832d5eb3f6ddc6ac/tests/ui/unit/filters.tests.js#L48-L59